resources/Materials/TestSuite/_options.mtlx

```diff
@@ -78,20 +78,20 @@
       <parameter name="specularEnvironmentMethod" type="integer" value="1" />
 
       <!-- Suggested radiance IBL file path -->
-      <parameter name="radianceIBLPath" type="string" value="resources/Images/Environments/san_giuseppe_bridge.hdr" />
+      <parameter name="radianceIBLPath" type="string" value="resources/Lights/san_giuseppe_bridge.hdr" />
 
       <!-- Suggested irradiance IBL file path -->
-      <parameter name="irradianceIBLPath" type="string" value="resources/Images/Environments/san_giuseppe_bridge_diffuse.hdr" />
+      <parameter name="irradianceIBLPath" type="string" value="resources/Lights/irradiance/san_giuseppe_bridge.hdr" />
 
       <!-- Transforms UVs of loaded geometry -->
-      <parameter name="transformUVs" type="matrix44" value="1.0f, 0.0f, 0.0f, -0.235f, 0.0f, -1.0f, 0.0f, 1.0f, 0.0f, 0.0f, 1.0f, 0.0f, 0.0f, 0.0f, 0.0f, 1.0f" />
+      <parameter name="transformUVs" type="matrix44" value="1.0f, 0.0f, 0.0f, 0.0f, 0.0f, -1.0f, 0.0f, 0.0f, 0.0f, 0.0f, 1.0f, 0.0f, -0.235f, 1.0f, 0.0f, 1.0f" />
```

Comment:
This is the extra uv transform we did to align with OSL geometry. It's column order and now the matrix library is row order so it's transposed.